binding/react-native/ios/Porcupine.swift

```diff
@@ -10,6 +10,12 @@
 //
 
 import PvPorcupine
+import Foundation
+
+enum PvPorcupineError: Error {
+    case RuntimeError(_ message: String)
```

Comment:
same comment as on Android. We should try to find a way to maintain custom exceptions all the way up.